content/faq/bookmarks/how-to-import/index.ru.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Как импортировать метки?
+description: ""
+updated: "2024-06-12"
+
+taxonomies:
+  faq: ["Метки и Треки"]
+
+extra:
+  order: 20
+---
+
+Вы можете импортировать метки отправленные из Organic Maps или сторонних приложений, если они позволяют экспортировать метки в форматах KML, KMZ, GPX:
+
+1. На Android, вы можете импортировать метки из некоторых других приложений, включая Maps.Me. Откройте метки (кнопка со звёздочкой), нажмите "Импорт меток", и выберите диск или папку, где будут KML, KMZ, GPX файлы.
+
+2. Отправьте себе на почту файл с метками (в формате KML, KMZ, GPX). Вы также можете использовать приложения для обмена сообщениями или загрузить файл из облачного хранилища, например, iCloud или Google Диск.
```

Comment:
```suggestion
2. Отправьте себе на почту файл с метками (в формате KML, KMZ, KMB, GPX). Вы также можете использовать приложения для обмена сообщениями или загрузить файл из облачного хранилища, например, iCloud или Google Диск.
```
Also should be mass-replaced.

Reply:
Added `KMB` to the list of supported formats all over the FAQ.